test/directive/ng_form_spec.dart

```diff
@@ -553,6 +553,51 @@ void main() {
         expect(form).not.toHaveClass('ng-required-invalid');
       });
 
+      it('should validate all the models when form.validate() is called',
+        (TestBed _, Scope scope) {
+
+        scope.context['required'] = true;
+        var formElement = _.compile('<form name="myForm">'
+                    '<input type="text" ng-model="one" probe="a" required />'
+                    '<input type="text" ng-model="two" probe="b" required />'
+                    '<input type="text" ng-model="three" probe="c" required />'
+                  '</form>');
+        scope.apply();
+
+        NgForm form    = _.rootScope.context['myForm'];
```

Comment:
style nitpic: I am against '=' aligning, and it doesn't match the rest of the code.